drivers/interrupt_controller/Kconfig

```diff
@@ -45,6 +45,13 @@ config LEON_IRQMP
 	help
 	  GRLIB IRQMP and IRQAMP
 
+config NUCLEI_ECLIC
+	bool "Extended Core Interrupt Controller (eclic)"
+	default y
```

Comment:
enable based on dt-compatible being enabled (see e.g. pinctrl or uart driver)

Reply:
I got it.
Modify the default line with dt_compat_enabled function.